### PR TITLE
TH-256: Origins > Add static footer component

### PR DESCRIPTION
**Pull request by eihabkhan1**
## JIRA Ticket

Ticket: [TH-256](https://youcanshop.atlassian.net/browse/TH-256).

## QA Steps

* [ ] Go to the homepage of your storefront
* [ ] Ensure that the footer abides by the [design](https://do3bol.netlify.app/)
* [ ] Ensure that the footer is responsive

## Note

Leave empty when you have nothing to say.

[TH-256]: https://youcanshop.atlassian.net/browse/TH-256?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: themes/origins/styles/base/_breakpoints.scss
Comment (eihabkhan1) [inline themes/origins/styles/base/_breakpoints.scss:1]: The breakpoints here are from: https://youcanshop.atlassian.net/wiki/spaces/ENG/pages/923664386/Tech+used#Breakpoints which are the same breakpoints used by [Tailwind](https://tailwindcss.com/docs/screens)

File: themes/origins/styles/footer.scss
Comment (ibrilBadreddine) [inline themes/origins/styles/footer.scss:1]: To keep the layout assets organized, rename this file to "layout-footer.scss", just like "layout-header.scss"

File: themes/origins/styles/base/_breakpoints.scss
Comment (ibrilBadreddine) [inline themes/origins/styles/base/_breakpoints.scss:1]: cool, but if u look at [do3bol](https://github.com/ibrilBadreddine/do3bol-theme/blob/88e49f154f74ba12306427ef3f0a95649f5065d3/src/views/components/layouts/style.scss#L262), the most frequently used breakpoints were 766px and 1180px, maybe there are more, but that’s what comes to mind

File: themes/origins/styles/footer.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  .content {
+    display: grid;
+    grid-template-columns: 30% 70%;
+    border-top: 1px solid var(--color-gray-50);
+    border-bottom: 1px solid var(--color-gray-50);
+    padding: 3em 10%;
+
+    .store-brand {
+      img {
+        max-width: 80px;
```
Comment (ibrilBadreddine) [inline themes/origins/styles/footer.scss:13]:
```SCSS
// <img src="" alt="" width="60" height="60" />
```
```suggestion
       width: 60px; 
       object-fit: cover;
```

File: themes/origins/styles/footer.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      }
+    }
+
+    .links-box {
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      gap: 1em;
```
Comment (ibrilBadreddine) [inline themes/origins/styles/footer.scss:20]:
```suggestion
      gap: var(--gap-lg);
```

File: themes/origins/styles/footer.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      align-items: start;
+
+      .navigation {
+        display: grid;
+        gap: 1em;
```
Comment (ibrilBadreddine) [inline themes/origins/styles/footer.scss:25]:
```suggestion
      gap: var(--gap-lg);
```

File: themes/origins/styles/footer.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+        .links {
+          display: grid;
+          gap: 0.5em;
```
Comment (ibrilBadreddine) [inline themes/origins/styles/footer.scss:29]:
```suggestion
      gap: var(--gap-md);
```

File: themes/origins/styles/footer.scss [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+            border-radius: var(--radius-sm);
+          }
+        }
+      }
+    }
+  }
+
+  .copyright {
+    padding: 2em;
+    font-size: 13px;
```
Comment (ibrilBadreddine) [inline themes/origins/styles/footer.scss:44]:
```suggestion
    font-size: var(--text-sm);
```

File: themes/origins/styles/footer.scss [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    .content {
+      padding: 3em 1.5em;
+
+      .links-box {
+        grid-template-columns: 1fr;
+        gap: 2em;
+
+        .navigation {
+          p {
+            font-size: 15px;
```
Comment (ibrilBadreddine) [inline themes/origins/styles/footer.scss:86]:
```suggestion
            font-size:  var(--text-lg);
```

**Review by ibrilBadreddine — APPROVED** [review]

**Pull request merged**